src/ruby/Exam4Program2.java

```diff
@@ -0,0 +1,47 @@
+package ruby;
+/*Program 2: WAP to find two String are identical or not
+
+input 1: 
+String s1 = "technoCredits";
+String s2 = "teChnoCreditS";
+
+output 1: both string are not identical
+
+input 2:
+String s1 = "pune";
+String s2 = "pune";
+
+output 2: both string are identical*/
+
+public class Exam4Program2 {
+	void CompareString(String s1, String s2) {
+		boolean flag = false;
+		int count = 0;
+		for (int i = 0; i < s1.length(); i++) {
+			if (s1.length() == s2.length()) {
+				flag = true;
+				if (s1.charAt(i) != s2.charAt(i)) {
+					count++;
+				}
```

Comment:
you should  put break, as you know now char is not matching then you can break the loop